sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/console.scala

```diff
@@ -17,58 +17,36 @@
 
 package org.apache.spark.sql.execution.streaming
 
-import org.apache.spark.internal.Logging
```

Comment:
can you move this file into the sources subdirectory to make it consistent with other v2 sources?

Reply:
in fact this file can be merged into the ConsoleWriter.scala. The combined file will be named console.scala

Reply:
I can do this in a followup PR. It's not as simple as just moving it; we have to add an alias so that .format("org.apache.spark.sql.execution.streaming.ConsoleSinkProvider") continues to work.

Reply:
argh. okay. later then.